### PR TITLE
[SPARKR] [HOTFIX] Disable flaky SparkR package build test

### PR DESCRIPTION
**Pull request by shivaram**
See https://github.com/apache/spark/pull/9390#issuecomment-157160063 and https://gist.github.com/shivaram/3a2fecce60768a603dac for more information

### CODE REVIEW

File: core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala [hunk 1]
```diff
@@ -369,6 +369,9 @@ class SparkSubmitSuite
     }
   }
 
+  // TODO(SPARK-9603): Building a package is flaky on Jenkins Maven builds.
+  // See https://gist.github.com/shivaram/3a2fecce60768a603dac for a error log
+  /*
   test("correctly builds R packages included in a jar with --packages") {
```
Comment (andrewor14) [inline core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala]: looks OK, but you can just do `ignore` here instead

Comment (JoshRosen) [inline core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala]: +1 on preferring ignore.

Comment (shivaram) [inline core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala]: Ah ok. I was just reverting the commit in #9390 which removed this comment

**Comment by shivaram** [comment]
Good to merge ?

**Comment by andrewor14** [comment]
Merging into master.

**Pull request closed without merging**